2016-08-21-Crash-Reporter.md

```diff
@@ -0,0 +1,35 @@
+### The Tessel Crash Reporter service
+
+The Tessel 2 [crash reporter](http://crash-reporter.tessel.io) is a web service which automatically collects anonymized crash reports when a crash is detected in the Tessel 2's command line interface (CLI). Given that the CLI is one of the primary ways of interacting with your Tessel it is important that the CLI be extremely stable and a joy to use.
+
+This project started during the early days of the CLI. We wanted to make sure that the team was proactively looking at, and fixing issues that users could potentially run into. We wanted the onboarding experience was as smooth as possible for new users.
```

Comment:
"could potentially" is redundant, and I think overly tentative. How about "...and fixing issues as users encountered them"